openshift-ci/Dockerfile.tools

```diff
@@ -1,4 +1,4 @@
-FROM centos:7 as build-tools
+FROM centos:8 as build-tools
```

Comment:
if you use centos:8 why do you need to update git manually?

Reply:
where did I update git "manually" ?

Reply:
the `tools` image will have an updated git is what I meant

Reply:
height of best practices in git commit messages :P